env0/resource_environment.go

```diff
@@ -140,94 +218,44 @@ func resourceEnvironment() *schema.Resource {
 				Type:        schema.TypeList,
 				Description: "terraform and environment variables for the environment",
 				Optional:    true,
+				Elem:        configurationSchema,
+			},
+			"without_template_settings": {
+				Type:         schema.TypeList,
+				Description:  "settings for creating an environment without a template. Is not imported when running the import command",
+				Optional:     true,
+				MinItems:     1,
+				MaxItems:     1,
+				ExactlyOneOf: []string{"without_template_settings", "template_id"},
+				Elem: &schema.Resource{
+					Schema: getTemplateSchema("without_template_settings.0."),
+				},
+			},
+			"sub_environment_configuration": {
+				Type:        schema.TypeList,
+				Description: "the subenvironments for a workflow enviornment. Template type must be 'workflow'. Must match the configuration as defined in 'env0.workflow.yml'",
+				Optional:    true,
```

Comment:
This diff is a bit messed up in github.
I recommend reviewing the file instead.